tests/configs/config-wrapper-zeroize-memset.h

```diff
@@ -0,0 +1,31 @@
+/* mbedtls_config.h wrapper that defines mbedtls_platform_zeroize() to be
+ * memset(), so that the compile can check arguments for us.
+ * Used for testing.
+ */
+/*
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include "mbedtls/mbedtls_config.h"
```

Comment:
Sorry I didn't mention this earlier, but the config wrapper format is inconvenient: we should use `MBEDTLS_USER_CONFIG_FILE` instead. See https://github.com/Mbed-TLS/mbedtls/pull/7961/commits/cf4fe58fd0ff3d6cb0624e7de76f144710027d2b — I'd forgotten that this isn't merged yet and so the file in `development` isn't a good example. Please follow the new format which is composable.

Reply:
Sorry. As is often the case, I encountered some side issues while dealing with the main one. It only took me a size-s time and none of the side issues were major so it didn't seem to be worth splitting.

Reply:
Yes, but... the preliminary commits are preparatory, and improves things generally, so could easily be separate PRs. They would be reviewed and merged quickly, improving the overall quality of the library... you know the drill